hpx/hpx_fwd.hpp

```diff
@@ -1453,39 +1437,6 @@ namespace hpx
         naming::id_type const& id);
 
     ///////////////////////////////////////////////////////////////////////////
-    /// \fn std::string get_locality_name()
-    ///
-    /// \brief Return the name of the locality this function is called on.
-    ///
-    /// This function returns the name for the locality on which this function
-    /// is called.
-    ///
-    /// \returns  This function returns the name for the locality on which the
-    ///           function is called. The name is retrieved from the underlying
-    ///           networking layer and may be different for different parcelports.
-    ///
-    /// \see      \a future<std::string> get_locality_name(naming::id_type const& id)
-    HPX_API_EXPORT std::string get_locality_name();
-
-    /// \fn future<std::string> get_locality_name(naming::id_type const& id)
-    ///
-    /// \brief Return the name of the referenced locality.
-    ///
-    /// This function returns a future referring to the name for the locality
-    /// of the given id.
-    ///
-    /// \param id [in] The global id of the locality for which the name should
-    ///           be retrieved
-    ///
-    /// \returns  This function returns the name for the locality of the given
-    ///           id. The name is retrieved from the underlying networking layer
-    ///           and may be different for different parcel ports.
-    ///
-    /// \see      \a std::string get_locality_name()
-    HPX_API_EXPORT future<std::string> get_locality_name(
-        naming::id_type const& id);
-
-    ///////////////////////////////////////////////////////////////////////////
```

Comment:
Those are not used anywhere and locality name doesn't make lots of sense without a defined parcelport.